app/controllers/api/users.php

```diff
@@ -415,6 +415,23 @@ function createUser(string $hash, mixed $hashOptions, string $userId, ?string $e
             }
         }
 
+        switch ($providerType) {
+            case 'email':
+                $validator = new Email();
+                if (!$validator->isValid($identifier)) {
+                    throw new Exception(Exception::GENERAL_INVALID_EMAIL);
+                }
+                break;
+            case 'sms':
+                $validator = new Phone();
+                if (!$validator->isValid($identifier)) {
+                    throw new Exception(Exception::GENERAL_INVALID_PHONE);
+                }
+                break;
+            default:
+                throw new Exception(Exception::PROVIDER_INCORRECT_TYPE);
```

Comment:
We don't want to throw for `push`, let's just have an empty case for it for now

Reply:
added case for push